introduction.md

```diff
@@ -1,3 +1,2 @@
-On my way to becoming a GitHub Master.
-
-My favorite Code School path in your introduction? Ruby, JavaScript, HTML & CSS, iOS or Electives
+ * Git
+ * JavaScript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?